e2e_tests/tests/requirements.txt

```diff
@@ -9,3 +9,5 @@ tensorflow==2.4.1
 pandas
 pyyaml
 docker
+boto3
```

Comment:
where does this come from?

Reply:
removed. my IDE was complaining at some point but seems to have stopped after invalidating caches and restarting.